### ISSUE TITLE
feat: use Bridgetown CLI directly instead of Yarn?

### ISSUE THREAD

**Issue by jaredcwhite** [bridgetownrb/bridgetown#280]
I put a question mark at the end there because I'm not entirely sold on the idea myself, but I've been musing on the idea of making the `bridgetown` command itself handle the common use cases we're currently relying on `yarn` to do. Think:

* `bridgetown start` (or `dev`)
* `bridgetown deploy`
* `bridgetown frontend:compile`
* etc.

I also did a personal hackathon a couple of days ago to try out Rake/Roda and in the processes essentially recreated Concurrently/Foreman/etc. in order to run multiple processes in parallel from a single Ruby script (Bridgetown watch, Webpack, and Browsersync). I like the idea of removing dependencies there and handling all the processes directly. But `yarn` has a lot of institutional awareness already and it's easy to tweak and add your own scripts in `package.json`. On the other hand…we're a Ruby tool, so why are using Yarn instead of, oh I dunno, Rake or [Toys](https://github.com/dazuma/toys) or something?

**Comment by jaredcwhite** [comment]
Earlier today prototyped `bridgetown` handling Rakefile tasks just like how `rails` does these days, and it's pretty nifty…thinking it might be the right call to move in that direction.

**Comment by jaredcwhite** [comment]
`bin/bridgetown start` etc. has been merged into v1.0 alpha. Closing.
